src/schema/v2/auction_result.ts

```diff
@@ -141,6 +141,39 @@ const AuctionResultType = new GraphQLObjectType<any, ResolverContext>({
         }
       },
     },
+    performance: {
+      type: new GraphQLObjectType({
+        name: "AuctionLotPerformance",
+        fields: {
+          mid: {
+            type: GraphQLString,
+            description: "Percentage performance over mid-estimate",
+            resolve: ({
+              hammer_price_cents,
+              high_estimate_cents,
+              low_estimate_cents,
+            }) => {
+              if (
+                hammer_price_cents &&
+                high_estimate_cents &&
+                low_estimate_cents
+              ) {
+                return (
+                  Math.round(
+                    ((hammer_price_cents -
+                      (low_estimate_cents + high_estimate_cents) / 2) /
+                      hammer_price_cents) *
+                      100
+                  ) + "%"
```

Comment:
Also, I think intermediate named variables could be helpful here from a readability/maintenance perspective. 

```suggestion
                const midEstimate = (low_estimate_cents + high_estimate_cents) / 2)
                const delta = hammer_price_cents - midEstimate
                return (
                  Math.round(
                    (delta / midEstimate) *
                      100
                  ) + "%"
```

(My formatting here is probably not correct)